src/main/java/Greet.java

```diff
@@ -0,0 +1,79 @@
+public class Greet {
```

Comment:
Perhaps the naming of this class should be named something differently, since not every method in this class is a form of greeting for the user. (eg addTask(), printGoodbyeMessage() ). Otherwise, i think that creating this class for this ip is a great idea.